### DISCUSSION TITLE
where the migrate guide from v1 to v2 ?

### DISCUSSION THREAD

**Discussion by whatwg6** [serverless/serverless#10300] [Q&A] [answered]
Sorry I did not find the guide about migrate from v1 to v2 , anyone help me.

**Comment by pgrzesik** [comment, marked as answer]
Hello,

there's no dedicated migration guide, but you can see the following links for specific details:
- [Release blog post](https://www.serverless.com/blog/serverless-framework-v2)
- [Release notes](https://github.com/serverless/serverless/releases/tag/v2.0.0)
